src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2617,11 +2617,12 @@ private static string CreateHostStringHelper(string str, ushort idx, ushort end,
             if (loopback)
             {
                 flags |= Flags.LoopbackHost;
-            }
-            if(linkLocalAddress)
+            }            
```

Comment:
There seems to be extra whitespace (space or tabs) after this closing brace.

Reply:
done